docs/data-collection.md

```diff
@@ -162,6 +162,11 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // cam_app_workload_migrations tracks number of app workload migrations
   // by current state. Tracked migration states are idle, running, completed, and failed.
   '{__name__="cam_app_workload_migrations"}',
+  // cluster:apiserver_current_inflight_requests:sum:max_over_time:1m gives maximum number of requests in flight
+  // over a 1-minute window. This metric is a constant 4 time series that monitors concurrency of kube-apiserver and
+  // openshift-apiserver with request type which can be either 'mutating' or 'readonly'.
+  // We want to have an idea of how loaded our api server(s) are globally.
+  '{__name__="cluster:apiserver_current_inflight_requests:sum:max_over_time:1m"}',
```

Comment:
What’s the scrape interval for the apiserver? 1m range query might fail if there is not at least 3x scrape intervals in 1min. I would suggest to bump it at least to 2m if not 5m.